cmd/nerdctl/container/container_prune.go

```diff
@@ -58,29 +62,29 @@ func containerPruneAction(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
-	client, ctx, cancel, err := newClient(cmd)
+	client, ctx, cancel, err := nerdClient.NewClient(cmd)
 	if err != nil {
 		return err
 	}
 	defer cancel()
 
-	return containerPrune(ctx, cmd, client)
+	return Prune(ctx, cmd, client)
 }
 
-func containerPrune(ctx context.Context, cmd *cobra.Command, client *containerd.Client) error {
+func Prune(ctx context.Context, cmd *cobra.Command, client *containerd.Client) error {
 	containers, err := client.Containers(ctx)
 	if err != nil {
 		return err
 	}
 
 	var deleted []string
 	for _, container := range containers {
-		err = removeContainer(ctx, cmd, container, false, true)
+		err = containerUtils.RemoveContainer(ctx, cmd, container, false, true)
 		if err == nil {
 			deleted = append(deleted, container.ID())
 			continue
 		}
-		if errors.As(err, &statusError{}) {
+		if errors.As(err, &common.StatusError{}) {
```

Comment:
> I am wondering if we should merge cmd/utils and pkg/util*, WDYT ?

I think it might be okay to have a minimum/lightweight `cmd/utils` that only relates to cobra cli or flag process. But I agree that most task/command specific logics should be on the `pkg` side. (No necessarily done altogether in one PR though :)).